packages/amplify_core/lib/src/types/datastore/hub/hub_event_element_with_metadata.dart

```diff
@@ -35,7 +35,7 @@ class HubEventElementWithMetadata<M extends Model> extends HubEventElement<M> {
   final int version;
 
   /// The last time the model was updated locally, in seconds since epoch.
-  final int lastChangedAt;
+  final int? lastChangedAt;
```

Comment:
This is part of the public API so it is technically a breaking change. Was this intended to be part of the public API? Is this a class that customers use (is there a use case in which they might use it)?

Ideally we should try to find a way to make this non-breaking. If that is not feasible and this was never intended to be public was can discuss deprecating this or marking it as internal.